pkgs/applications/misc/gpg-mdp/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchurl, stdenv, ncurses, gnupg }:
+
+stdenv.mkDerivation {
+  # mdp renamed to gpg-mdp because there is a mdp package already.
+  name = "gpg-mdp-0.7.4";
```

Comment:
Please factor out the version and re-use it in the url, to prevent them getting out of sync.